src/aks-preview/create-5.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+export SUBSCRIPTION_ID=c1089427-83d3-4286-9f35-5af546a6eb67
+export LOCATION=eastus2euap
+export RESOURCE_GROUP_NAME=weinongw-${LOCATION}
+export AKS_NAME=weinong-p-
+export VNET_NAME=weinongw-vnet
+export AKS_SUBNET_NAME=aks
+export AKS_VERSION=1.14.6
+export AKS_SPN_CLIENT_ID=13bec9da-7208-4aa0-8fc7-47b25e26ff5d
+export AKS_SPN_SECRET=2aQUC72zjh_C8+C.]kSdfKGjrH52DZ[Z
```

Comment:
hard code secret can not get CI job CredScan pass.

Please don't hard code any secret or password.

Reply:
sorry about that. i removed it and reset the password. :(